### PR TITLE
Adding feature: Hide watched videos on Youtube homepage

### PR DESCRIPTION
**Pull request by stalha97**
Just as this extension hides watched videos on subscription page. With a bit of modification in files, the same feature is extended to homepage of Youtube as well. The UI buttons appear on homepage to enable/disable "hide watched videos" and the other button which makes all the videos on the webpage "watched". And the small icon on individual videos as well, where when you click, that video gets added to "watched" list.

### CODE REVIEW

**Comment by OsaSoft** [comment]
Hi and thanks for the PR!

Ive looked at the code and it seemed weird it would work. I then tested it locally and it really doesnt actually seem to do anything. Please check you have everything pushed

(I also took the liberty to rebase your branch onto develop)

**Comment by stalha97** [comment]
Hi, thanks for your reply. I have tested it again locally and it is working. I checked the code on develop branch and apparently there have been a lot of changes than when I programmed on it and I think that messed up with the work I did. I am attaching an image of what I am seeing with what I originally coded on.

image

File: manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "short_name": "Better Subs",
   "manifest_version": 2,
   "name": "Better Subscriptions for YouTube™",
-  "version": "0.13.1.1",
+  "version": "0.14",
```
Comment (OsaSoft) [inline manifest.json:6]: pls change back, version is updated before release (though Ill probably change this process in the future)

File: pageHandler.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ function initExtension() {
         "subscriptions": "/feed/subscriptions",
         "video": "/watch",
         "channel": "/videos",
+		"home": ""
```
Comment (OsaSoft) [inline pageHandler.js:13]: please fix indents. Tabs -> spaces

File: pageHandler.js [hunk 3]
```diff
@@ -29,6 +30,9 @@ function initExtension() {
                     break;
                 case PAGES.video:
                     onVideoPage();
+                    break
+				case PAGES.home:
```
Comment (OsaSoft) [inline pageHandler.js:34]: please fix indents. Tabs -> spaces

**Review by OsaSoft — CHANGES_REQUESTED** [review]
Ive ran it again locally and for some reason it seems to have broken the functionality added in the previous MR (only show the Mark as watched button on thumbnail hover), it now doesnt show on hover. (at least for me testing using Chrome, havent tested in Firefox)

If you can fix that great, if not no big deal and Ill still merge the PR and just disable that feature for now, as it seems a bit unreliable in the way its currently implemented.

Thanks!

**Comment by OsaSoft** [comment]
ok, since the PR has been inactive for some time and requested changes are minor, I shall do them manually. But for future PRs, they will be closed after some time of inactivity

**Pull request merged**